CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+### citus v11.1.3 (October 14, 2022) ###
+
+* Adds support for PostgreSQL 15.0
+
+* Fixes a bug in `ALTER EXTENSION citus UPDATE`
+
+* Fixes a bug that causes crashes in `ALTER ROLE role PASSWORD NULL`
```

Comment:
After some discussion with PR author @naisila , I will change this into `Fixes a bug that causes a crash with empty/null password`